ansible/roles/etcd/tasks/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+# tasks file for etcd
+
+- name: install etcd from centos extras repository
+  yum: name=etcd state=installed
```

Comment:
what about when a newer version releases? `state=installed` will not update the packages